camera_name: d_phoxi_m_camera
robot_name: d_bot
eye_on_hand: false
robot_base_frame: workspace_center
robot_effector_frame: d_bot_gripper_base_link
robot_effector_tip_frame: d_bot_gripper_tip_link
camera_parent_frame: workspace_center
marker_map: aruco-26-70x70-5
planarity_tolerance: 0.001
initpose: [0.10,  0.10, 0.35, 0, 90, -90]
keyposes:
  - [ 0.20, -0.14, 0.16, 30, 25, -90]
  - [ 0.05, -0.14, 0.16, 30, 25, -90]
  - [-0.10, -0.14, 0.16, 30, 25, -90]

  - [-0.15,  0.10, 0.21, 30, 25, -90]
  - [ 0.00,  0.05, 0.21, 30, 25, -90]
  - [ 0.15, -0.05, 0.21, 30, 25, -90]
